cmd/prometheus-config-reloader/main.go

```diff
@@ -109,6 +110,8 @@ func main() {
 	runtimeInfoURL := app.Flag("runtimeinfo-url", "URL to check the status of the runtime configuration").
 		Default("http://127.0.0.1:9090/api/v1/status/runtimeinfo").URL()
 
+	serverBasicAuth := app.Flag("server-basic-auth", "Basic Auth credentials for the prometheus server. The format should be 'username:password'").Default("").String()
```

Comment:
We can't have the password in clear-text.

Reply:
> We can't have the password in clear-text.

So we should use a config file to save the basic auth info in `reloader` ? like we mount an additional file into contaienr, and the flag may like `server-pod-credentials-config-file`.